### DISCUSSION TITLE
How to attend online?

### DISCUSSION THREAD

**Discussion by yuzeng2333** [cucapra/latte21#46] [General]
I have difficulty attending it. I clicked the "Online Venue" on ASPLOS website, but the clowdr website says "Sorry, we couldn't find that page." Does anyone have any idea how to fix this?

**Comment by rachitnigam** [comment]
Can you go to app.clowdr.org/ and see if ASPLOS shows up? Pinging @sampsyo if there are more problems.

**Reply by sampsyo** [reply]
If it still doesn't work, @yuzeng2333, can you email me (asampson@cs.cornell.edu) and I'll send you the Zoom link directly?

**Reply by sampsyo** [reply]
(You also probably want to make sure you're registered for ASPLOS if you're not already.)

**Reply by yuzeng2333** [reply]
That website asks for invite code. Do you have one? @sampsyo

**Reply by sampsyo** [reply]
@yuzeng2333, are you registered for ASPLOS? I think it's instantaneous: https://web.cvent.com/event/6259afee-6594-4456-86a0-2a22fbfc47b8/summary

**Reply by yuzeng2333** [reply]
Yes, I have registered for both ASPLOS and clowdr. I'll send you an email. Thanks!

**Reply by sampsyo** [reply]
Got it. The short answer is that ASPLOS should have sent you a personalized Clowdr invitation link that lets you bypass the whole code thing.

**Reply by yuzeng2333** [reply]
Thanks! I have found the email sent by clowdr.
